wg-naming/recommendations/template.md

```diff
@@ -0,0 +1,17 @@
+# Recommendation: Old Term -> New Term
+
+**Status:** Accepted|Draft|Other
+
+## Recommendation 
```

Comment:
I'm wondering if "Suggested Alternatives" as a heading might make this easier to search for.  At least for me, "<term> alternative" and "<term> replacement" yielded more relevant content on Google, Bing and DuckDuckGo

Reply:
readability suggestion: kind of adjacent to Aaron's point - we've got "Recommendation: <>" as H1 and "Recommendation" as H2 right under, it would probably be easier to read if this was a different heading. Or am I misunderstanding and this heading is the actual recommendation itself?

Reply:
I think that's a great suggestion, accepted :)